sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_lro.py

```diff
@@ -125,28 +132,56 @@ def id(self):
         return self._current_body.job_id
 
 
-class AnalyzeHealthcareEntitiesLROPoller(LROPoller):
+class AnalyzeHealthcareEntitiesLROPoller(LROPoller, Generic[PollingReturnType]):
+
+    def polling_method(self):
```

Comment:
do users need to access this?

Reply:
no, this was for typing basically. It's inherited from `LROPoller`, but since I added metadata on my pollers + polling methods (metadata on polling methods isn't really exposed to users, but is used to pass info to the poller), I'm using the polling method to let mypy know that "yep, my poller / polling method really has these metadata attributes"